test/luatest_helpers/vtest.lua

```diff
@@ -109,6 +159,59 @@ local function storage_new(g, cfg)
     end
 end
 
+--
+-- Apply the config to all vshard storages in the cluster.
+--
+local function storage_cfg(g, cfg)
+    -- No support yet for dynamic node addition and removal. Only reconfig.
+    local fids = {}
+    local storages = {}
+    -- Map-reduce. It should make reconfig not only faster but also not depend
+    -- on which order would be non-blocking. For example, there might be a
+    -- config which makes the master hang until some replica is configured
+    -- first. When all are done in parallel, it won't matter.
+    for _, storage in pairs(g.cluster.servers) do
+        if storage.vtest and storage.vtest.is_storage then
+            table.insert(storages, storage)
+            table.insert(fids, storage:exec(function(cfg)
+                local f = fiber.new(vshard.storage.cfg, cfg, box.info.uuid)
+                f:set_joinable(true)
+                return f:id()
+            end, {cfg}))
+        end
+    end
+    local errors = {}
+    for i, storage in pairs(storages) do
```

Comment:
It's better to use here ipairs. pairs doesn't give order guarantee (It works for now but it doesn't mean that it will work in future).

Reply:
Order does not matter here. I can send and collect responses in any order because they are executed in parallel all at the same time anyway.